icubam/backoffice/templates/login.html

```diff
@@ -1,42 +1,76 @@
-<!--
-This is a starter template page. Use this page to start your new project from
-scratch. This page gets rid of all links and provides the needed markup only.
--->
-<html lang="en">
+<!DOCTYPE html>
+<html>
 <head>
   <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>ICUBAM-admin | Log in</title>
+  <!-- Tell the browser to be responsive to screen width -->
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta http-equiv="x-ua-compatible" content="ie=edge">
 
-  <title>ICUBAM</title>
-
-  <!-- Font Awesome Icons -->
-  <link rel="stylesheet" href="plugins/fontawesome-free/css/all.min.css">
+  <!-- Font Awesome -->
+  <link rel="stylesheet" href="../../plugins/fontawesome-free/css/all.min.css">
+  <!-- Ionicons -->
+  <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css">
+  <!-- icheck bootstrap -->
+  <link rel="stylesheet" href="../../plugins/icheck-bootstrap/icheck-bootstrap.min.css">
   <!-- Theme style -->
-  <link rel="stylesheet" href="dist/css/adminlte.min.css">
+  <link rel="stylesheet" href="../../dist/css/adminlte.min.css">
   <!-- Google Font: Source Sans Pro -->
   <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700" rel="stylesheet">
 </head>
-<body class="hold-transition sidebar-mini">
- <div class="wrapper">
+<body class="hold-transition login-page">
+<div class="login-box">
+  <div class="login-logo">
+    <b>ICUBAM</b> admin</a>
+  </div>
+  <!-- /.login-logo -->
+  <div class="card">
+    <div class="card-body login-card-body">
+      <p class="login-box-msg">{{_("Sign in to start your session")}}.</p>
+      {% if error %}
+      <p class="login-box-msg text-danger">{{_("Authentication failed")}}.</p>
+      {% end %}
 
-  <!-- Custom styles for this template -->
-  <link href="signin.css" rel="stylesheet">
+      <form action="/login" method="post" class="form-signin">
+        <div class="input-group mb-3">
+          <!-- <input type="email" class="form-control" placeholder="Email" name="email"> -->
+          <input class="form-control" placeholder="Email" name="email">
```

Comment:
add required for some quick client side validation. Also autofocus (helps a lot in mobile)